GUIDE.md

```diff
@@ -0,0 +1,81 @@
+Pegaswitch Getting Started Guide
+================================
+A guide to getting your Switch from factory to running Pegaswitch.
+
+Requirements
+------------
+- A Nintendo Switch running FW 1.0.0 - 3.1.0
+	- For 1.0.0, you will also need a copy of [Puyo Puyo Tetris JP](https://www.amazon.com/gp/product/B01N4PKS4S/ref=oh_aui_detailpage_o00_s00?ie=UTF8&psc=1)
+- A PC running any OS capable of running Node.JS that is connected to a network your Switch can connect to
+
+Setting up the Environment
+--------------------------
+### Windows
+Windows users will need to setup the Windows Subsystem for Linux (WSL). Instructions on how to set that up can be found [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10). After WSL has been setup, continue with the Linux instructions, using the Ubuntu/Debian commands.
+### Linux
+You will need to install `git` and `nodejs` from your distributions package manager. Please note that the version of NodeJS installed must be 8.x.x or higher.
+- **Debian-based (Ubuntu, Mint, etc.)**: An alternate repository is required, as Ubuntu/Debian ships very old versions of NodeJS.
+```bash
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install nodejs git build-essential
+```
+- **Fedora**:
+```bash
+sudo dnf install nodejs npm git
+```
+- **Arch Linux**:
+```bash
+sudo pacman -S nodejs npm git base-devel
+```
+- **Other**: See the [installation guide for Node.JS](https://nodejs.org/en/download/package-manager/) to install Node.JS. Consult your distributions package manager to find packages for `git` and `nodejs` if the installation guide doesn't support it.
+### OS X/macOS
+You will need to install [Homebrew](https://brew.sh/) to download the necessary packages. After Homebrew is installed:
+```bash
+brew install node git
+```
+Setting up Pegaswitch
+---------------------
+1. In a terminal window, navigate to the location you want to install Pegaswitch to, and clone the Pegaswitch git repository using `git clone https://github.com/reswitched/pegaswitch.git`.
+2. Navigate into the new `pegaswitch` directory, and run `npm i`. This will install all the modules needed for Pegaswitch to function.
+3. Start Pegaswitch with `sudo node start --setuid $(id)`. Your user id can be found by running `id`, and searching the number next to your username (typically 1000, on a single-user system).
+	- If you are using Puyo Puyo Tetris or fakenews to access Pegaswitch, add `--webapplet` to this command. 
+
+Launching Pegaswitch
+--------------------
+4. Setup a connection to the Internet on your Switch, if you haven't already. Don't stay connected for too long without setting the DNS, otherwise your Switch may download an update.
```

Comment:
`Setup` -> `Set up`. Also, we should not be recommending that users ever connect to the internet for even the shortest amount of time. I'm pretty sure there's a way to get at the internet settings menu before connecting. I think I always did it by intentionally typing the wrong WPA2 passphrase so it would let me get at that menu.